apps/src/templates/accessible-dialogue.module.scss

```diff
@@ -6,19 +8,36 @@
   left: 0;
   background-color: #000;
   opacity: 0.6;
-  z-index: 1000;
+  z-index: 1250;
 };
 
 .modal {
   position: fixed;
   top: 10%;
   left: 50%;
   transform: translateX(-50%);
-  z-index: 1050;
+  z-index: 1350;
   width: 70%;
   max-width: 600px;
   background-color: #fff;
   border-radius: 4px;
   padding: 1rem;
   overflow: auto;
 };
+
```

Comment:
Added this formatting to the accessible dialog stylesheet instead of a specific one for this dialog because it seems to be a standard across most of our dialogs (a line above body test, buttons floated to the right, etc).